src/logic/operate.js

```diff
@@ -0,0 +1,18 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let res;
+  if (operation === 'X') {
+    res = Big(numberOne) * Big(numberTwo);
+  } if (operation === '÷') {
+    res = Big(numberOne) / Big(numberTwo);
+  } if (operation === '+') {
```

Comment:
- [ ] **[OPTIONAL]** I suggest that you use switch statement or (if.. else if blocks). The control flow of this function will step through each `if` condition even if the operation is already handled in the previous `if` condition.